src/collections/vec.rs

```diff
@@ -1983,26 +1983,23 @@ __impl_slice_eq1! { Vec<'a, A>, &'b [B] }
 __impl_slice_eq1! { Vec<'a, A>, &'b mut [B] }
 // __impl_slice_eq1! { Cow<'a, [A]>, Vec<'b, B>, Clone }
```

Comment:
I'm not sure why this is commented out, should this be implemented or not? If not, the `__impl_slice_eq1!` macro can be simplified a bit.